test/field_types/test_file_field.py

```diff
@@ -19,6 +22,14 @@ class ModelWithFile(MongoModel):
     secondary_upload = FileField(required=False)
 
 
+class ModelWithFileNoConnection(MongoModel):
+    upload = FileField()
+    secondary_upload = FileField(required=False)
```

Comment:
What is the significance of having two file fields? And doesn't required default to False anyway?